nabs/utils.py

```diff
@@ -0,0 +1,62 @@
+from ophyd.signal import DerivedSignal
+from ophyd.sim import SignalRO
+
+
+class InvertedSignal(DerivedSignal):
```

Comment:
starting to find more candidates for a `pcdsutils` module

Reply:
👍